src/test/java/uk/gov/hmcts/sptribs/document/bundling/BundlingServiceTest.java

```diff
@@ -262,37 +292,260 @@ void shouldReturnNullWhenNoBundles() {
 
         when(bundlingClient.createBundle(any(), any(), any())).thenReturn(bundleResponse);
 
-        Callback callback = new Callback(updatedCaseDetails, beforeDetails, CREATE_BUNDLE, true);
-        BundleCallback bundleCallback = new BundleCallback(callback);
-        //When
-        List<Bundle> result = bundlingService.createBundle(bundleCallback);
-        List<ListValue<Bundle>> resultList = bundlingService.buildBundleListValues(result);
+        final Callback callback = new Callback(updatedCaseDetails, beforeCaseDetails, CREATE_BUNDLE, true);
+        final BundleCallback bundleCallback = new BundleCallback(callback);
+
+        final List<Bundle> result = bundlingService.createBundle(bundleCallback);
+        final List<ListValue<Bundle>> resultList = bundlingService.buildBundleListValues(result);
 
-        //Then
         verify(bundlingClient).createBundle(any(), any(), any());
         assertThat(result).isEmpty();
         assertThat(resultList).isNull();
-    }*/
+    }
 
     @Test
     void shouldReturnNullWhenNoBundleFolders() {
```

Comment:
A new test case can be added to create bundled folder with values.

Reply:
I think these are covered in the parametrised tests `shouldCreateBundleListValues` and `shouldCreateBundle`

Reply:
yes, right (as these methods were used in create bundle). Is it possible to move/cover the null test as part of another parameterised method? (just to maintain consistency). It's ok if it is not doable.

Reply:
I have updated the tests for the default and empty/null folders to be included in the parametrised tests for `createBundle` in the test `shouldCreateBundle`. But was unable to move the `shouldReturnNullWhenNoBundles` into `shouldCreateBundleListValues`